src/modules/70-pipeline/components/PipelineSteps/Steps/ServiceNowApproval/ServiceNowApprovalRejectionCriteria.tsx

```diff
@@ -263,9 +261,9 @@ export function ServiceNowApprovalRejectionCriteria(props: SnowApprovalRejection
     if (!isEmpty(props.fieldList)) {
       // If the status list is non empty, initialise it so that status is by default a dropdown
       allowedFieldKeysToSet = props.fieldList.map(field => {
-        allowedValuesForFieldsToSet[field.key] = CDS_SERVICENOW_USE_METADATA_V2
+        allowedValuesForFieldsToSet[field.key] = !isEmpty(field.allowedValues)
```

Comment:
```suggestion
        allowedValuesForFieldsToSet[field.key] = setAllowedValuesOptions(field?.allowedValues ?? [])
```
isEmpty() check is not needed, optional check would be sufficient

Reply:
Thanks! 
removed